lib/ansible/plugins/callback/logentries.py

```diff
@@ -212,7 +213,7 @@ def __init__(self):
                                  'Disabling the Logentries callback plugin.')
 
         config_path = os.path.abspath(os.path.dirname(__file__))
-        config = ConfigParser.ConfigParser()
+        config = configparser.ConfigParser()
```

Comment:
<nod>  What I was wondering before is that I believe in python2 this is the equivalent of RawConfigParser and i python3 it's the equivalent of SafeConfigParser.  Just wondered if there's anything else that needed to be adjusted for that change.